mathics/core/evaluation.py

```diff
@@ -233,7 +233,8 @@ def __init__(
             definitions=None,
             output=None,
             format="text",
-            catch_interrupt=True
+            catch_interrupt=True,
+            use_unicode=True
```

Comment:
I think this is worng. This is a property of formatting, not evaluation.

Reply:
@rocky I agree, but I don't know where to place it.